picard/file.py

```diff
@@ -167,7 +167,15 @@ def _load_check(self, filename):
         if self.tagger.stopping:
             log.debug("File not loaded because %s is stopping: %r", PICARD_APP_NAME, self.filename)
             return None
-        return self._load(filename)
+        try:
+            # Try loading based on extension first
+            ret = self._load(filename)
+        except Exception:
```

Comment:
This is one of the few cases where I think catching a generic exception is ok. Fetching a `MutagenError` might be more specific, but there had been some cases where mutagen throws all kinds of exception (even though there have been a couple of upstream fixes for this recently).